platform/ios/docs/guides/Info.plist Keys.md

```diff
@@ -10,6 +10,21 @@ Mapbox-hosted vector tiles and styles require an API access token, which you can
 
 As an alternative, you can use `MGLAccountManager.accessToken` to set a token in code. See [our guide](https://www.mapbox.com/help/ios-private-access-token/) for some tips on keeping access tokens in open source code private.
 
+## MGLAccuracyAuthorizationDescription
+
+Set the Mapbox accuracy authorization description string as an element of `NSLocationTemporaryUsageDescriptionDictionary` to be used by the map to request authorization when the `MGLLocationManager.accuracyAuthorization` is set to `CLAccuracyAuthorizationReducedAccuracy`.
+
+Example:
+```
+<key>NSLocationTemporaryUsageDescriptionDictionary</key>
+	<dict>
+		<key>MGLAccuracyAuthorizationDescription</key>
+		<string>Mapbox requires your precise location to help you navigate the map.</string>
+	</dict>
+```
+
+Remove `MGLAccuracyAuthorizationDescription` if you want to control when to request for accuracy authorization.
```

Comment:
I think this section needs more color, as this section feels unclear to me. An example that illustrates this further would be ideal.